views/components/BarGraph/index.jsx

```diff
@@ -30,7 +30,7 @@ const BarGraph = ({ graph }) => {
   }
 
   return (
-    <ResponsiveContainer width="100%" height={500}>
+    <ResponsiveContainer width="99%" height={500}>
```

Comment:
interesting, any idea why that is the case? Do you have any reference material like a stack overflow post or documentation link?

Reply:
For version 2.9 of recharts the 99% setting was a hacky solution to this problem, with the latest version of recharts 2.10 this issue should have gone away. 
https://github.com/recharts/recharts/issues/172

I've updated the library and the issue is fixed [f8a91b6](https://github.com/AMP-SCZ/dpdash/pull/599/commits/f8a91b625c0f5138f0556f043e0b44015e8c11c4)